src/allotropy/parsers/appbio_quantstudio/appbio_quantstudio_builders.py

```diff
@@ -623,25 +624,23 @@ def build(reader: LinesReader) -> Data:
                     )
                 )
 
-            well.calculated_document = CalculatedDocumentBuilder.build_quantity(
-                well_item
-            )
-
-        endogenous_control = get_str(results_metadata, "Endogenous Control")
-        reference_sample = get_str(results_metadata, "Reference Sample")
+            well.calculated_document = build_quantity(well_item)
 
-        calc_doc_builder = CalculatedDocumentBuilder(
-            wells=wells,
-            experiment_type=header.experiment_type,
-            r_target=endogenous_control or "",
-            r_sample=reference_sample or "",
-        )
+        endogenous_control = get_str(results_metadata, "Endogenous Control") or ""
+        reference_sample = get_str(results_metadata, "Reference Sample") or ""
```

Comment:
Please don't include refactors with functional changes. I.e., leave these lines as is and only change the `calc_doc_builder` call.

Out of scope: `get_str` has a `default` param, but it's not very useful for type checking since it can be optional and the function returns `Optional[str]`. If we had a utility method like `get_str_or_default` where `default` was required and `str` was returned, we could use that here.